params/config.go

```diff
@@ -101,7 +102,7 @@ var (
 // that any network, identified by its genesis block, can have its own
 // set of configuration options.
 type ChainConfig struct {
-	ChainId *big.Int `json:"chainId"` // Chain id identifies the current chain and is used for replay protection
+	ChainID *big.Int `json:"chainID"` // Chain id identifies the current chain and is used for replay protection
```

Comment:
This one changes our genesis format, please revert. Yes, it's wrong, but it's part of the API, so if you uppercase it, Go will refuse to parse the jsons.

Reply:
Just to clarify, the field name is fine as you changed it, just the json tag needs to be reverted.

Reply:
Great ok - updated.